addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1928,18 +1940,25 @@ options.registry.SnippetMove = options.Class.extend({
     moveSnippet: function (previewMode, widgetValue, params) {
         const isNavItem = this.$target[0].classList.contains('nav-item');
         const $tabPane = isNavItem ? $(this.$target.find('.nav-link')[0].hash) : null;
+        const $buttons = this.$overlay.find('.o_overlay_move_options').find('we-button');
         switch (widgetValue) {
             case 'prev':
                 this.$target.prev().before(this.$target);
                 if (isNavItem) {
                     $tabPane.prev().before($tabPane);
                 }
+                if ($buttons.is('.fa-angle-up, .fa-angle-down')) {
```

Comment:
Instead of this, add a data-auto-scroll param on the buttons and check if the value is in `params` here.